src/themes/default/components/core/blocks/Header/MicrocartIcon.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div class="microcart-icon" @click="toggleMicrocart">
     <i class="material-icons md-18">shopping_cart</i>
+    <span class="minicart-count l-center-vh brdr-circle fs-10 weight-700 bg-gray c-white" v-cloak>{{totals.quantity}}</span>
```

Comment:
for 1-center-vh - you can use center-md middle-md classes from flexbox grid to achieve this behavior, http://flexboxgrid.com/
It's our fault that there are no CSS docs so i will merge it and correct by myself ;)